### PR TITLE
Add models for python grpc testing

### PR DESCRIPTION
**Pull request by ackizilkale**
# Eagle 2050
- Add new model tests
- Increase test timeout to 30 mins

Developed by @steph-en-m . I opened the PR since the tests were running into authorization issues in his PR.

### CODE REVIEW

**Review by HarmitMinhas96 — COMMENTED** [review]
These seems a bit excessive, do we need to test these many models?
If we know certain models share the same libs but the only difference is the weights, we shouldn't need to test all of them.
E.g. all the translation models?

**Comment by ackizilkale** [comment]
I agree. @steph-en-m is this the list you agreed with @cemigo114 ? For instance, FACEBOOK_TRANSLATION_MODELS is very long.

**Comment by steph-en-m** [comment]
We aren't testing all translation models.
I will remove some more models though to shorten the tests.

**Comment by steph-en-m** [comment]
Removed some of the models from tests. #99

File: tests/common.py [hunk 1]
```diff
@@ -19,13 +19,26 @@
...
 TOY_VIDEO_FILE_PATH = os.path.dirname(__file__) + "/assets/toy.mp4"
 
-ENGLISH_TEXT = (
-    "My spanish test is tomorrow morning. I don't feel like studying tonight, but I must study."
-)
-SPANISH_TEXT = "No me apetece nada estudiar esta noche."
-
 ENGLISH_AUDIO_URL = "https://samples.clarifai.com/english_audio_sample.mp3"
 
+TRANSLATION_TEST_DATA = {
```
Comment (HarmitMinhas96) [inline tests/common.py:24]: Given these are specific to the model predict test, maybe it shouldn't be in tests/common ?

I see there's other things in this file that aren't common either, but just staying within scope

Comment (steph-en-m) [reply tests/common.py:24]: yeah I see.
Do you suggest that I move them to the test_public_models_script or perhaps a separate file?

Comment (HarmitMinhas96) [reply tests/common.py:24]: I think either moving them into test_public_models_script would be good, because it has to import all of them anyways

Comment (steph-en-m) [reply tests/common.py:24]: sure let me do that

**Review by HarmitMinhas96 — APPROVED** [review]

File: tests/common.py (hunk 1 above)
Comment (ackizilkale) [reply tests/common.py]: I confirm all tests pass properly. But we will add them slowly to make sure all is scalable. @HarmitMinhas96  see [here](https://github.com/Clarifai/clarifai-python-grpc/blob/eagle2050-rebased/tests/public_models/test_public_models_predicts.py#L394) and [here](https://github.com/Clarifai/clarifai-python-grpc/blob/eagle2050-rebased/tests/public_models/test_public_models_predicts.py#L419) 

Are you cool with that @HarmitMinhas96  ? I will create tickets for those two tests so we add one each week.

I will also let Marco know what to synchronize across dev/staging/prod

**Comment by ackizilkale** [comment]
Oh there is also [this](https://github.com/Clarifai/clarifai-python-grpc/blob/eagle2050-rebased/tests/public_models/test_public_models_predicts.py#L122). These are big models. So I will increase this group slowly also.

**Review by HarmitMinhas96 — COMMENTED** [review]
For the skipped tests, maybe we can test just one model in each of them? And instead comment out all models except 1

**Comment by ackizilkale** [comment]
yeah that makes sense. Just pushed the code.

**Review by HarmitMinhas96 — APPROVED** [review]
Sorry for the lazy review style I've been doing here but one last comment:
It looks like lines 0 to 341 might be good to be in their own file in the public_models folder, and then the tests cases here only actually have to import a few of the aggregated variables. 

Though if you prefer not to do that, that's ok to

**Comment by HarmitMinhas96** [comment]
If you did the above mentioned refactor, then a lot of the vars in tests/common.py can also be moved since they're only there for the public models test

**Pull request merged**